scenes/game/local_game.gd

```diff
@@ -362,7 +362,7 @@ class Player:
 	var has_overdrive : bool
 	var set_aside_cards : Array[GameCard]
 	var deck_def : Dictionary
-	var gauge : Array[GameCard]
+	var gauge : Array
```

Comment:
this one should be safe to keep. If something not a GameCard gets put in the player gauge, it's going to get real bad real quick.
Anything that is putting stuff there should be getting it from something already marked GameCard like hand/discards